.github/workflows/continuous-integration.yml

```diff
@@ -117,11 +117,18 @@ jobs:
           CHANGED_FILE_PATHS: ${{ steps.changed-files.outputs.all_changed_files }}
 
       - name: Run unit tests
-        run: yarn test:unit ${APP_FOLDERS:+"{script,$APP_FOLDERS}/**/*.unit.spec.js?(x)"} --coverage
+        run: yarn test:unit --coverage ${APP_FOLDERS:+"{script,$APP_FOLDERS}/**/*.unit.spec.js?(x)"}
```

Comment:
Since we don't need to change this part anymore, can this change be reverted?

Reply:
Yes, I reverted it. To be clear, I'm keeping `--coverage` but I'm moving it to the end.